<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="chooseDirButton">
       <property name="text">
        <string>Choose Dir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dirLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="findFilesButton">
       <property name="text">
        <string>Find Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="foundFilesLabel">
       <property name="text">
        <string> :  Number of files found</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Single Threaded</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="singleThreadedButton">
            <property name="text">
             <string>Single Threaded</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="singleThreadedLabel">
            <property name="text">
             <string>Single threaded report</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Map reduced</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="mapReducedButton">
            <property name="text">
             <string>MapReduced</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mapReducedLabel">
            <property name="text">
             <string>Map reduced report</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
